ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml

```diff
@@ -23,6 +23,10 @@ base_images:
     name: stream8
     namespace: coreos
     tag: "8"
+  coreos_centos_stream9:
```

Comment:
be aware it does not use it currently as there was no image substitution in the building of ``project_image``